packages/blocks/tests/utils/from-theme-json/fromThemeJson.test.ts

```diff
@@ -0,0 +1,473 @@
+import { fromThemeJson } from '../../../src/utils/from-theme-json/fromThemeJson';
+import theme from './theme.test.json';
+
+describe('fromThemeJson()', () => {
+  it('returns an empty object when passed an empty theme.json object', () => {
+    const themeJson = {};
+    const expectedResult = {};
+    expect(fromThemeJson(themeJson)).toEqual(expectedResult);
+  });
+
+  it('returns a BlocksTheme object when passed a theme.json object with settings and styles', () => {
+    const themeJson = theme;
+
+    expect(fromThemeJson(themeJson)).toMatchInlineSnapshot(`
+      Object {
+        "fontFamilies": Object {
```

Comment:
Our schema is a more compact version of the theme.json with only the relevant parts. We might improve this on a use-case basis.